calibration/SPKIRB/CGINS-SPKIRB-00302__20170911.csv

```diff
@@ -1,4 +1,4 @@
 serial,name,value,notes
-302,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347] ",
-302,CC_offset,"[2147969425.9, 2147121996.9, 2147405792.6, 2147708863.6, 2147330032.5, 2147857974.5, 2147980509.1] ",
-302,CC_scale,"[2.01373417518e-07, 1.9315808239e-07, 2.0030879899e-07, 2.00056479715e-07, 1.83693724568e-07, 2.11280599416e-07, 209272490423e-07] ",
+302,CC_immersion_factor,"[1.368, 1.41, 1.365, 1.354, 1.372, 1.322, 1.347]", source_file SPKIR_OCR-507_SN_302_Calibration_Files_2017-09-11.zip > DI7302B.cal
+302,CC_offset,"[2147969425.9, 2147121996.9, 2147405792.6, 2147708863.6, 2147330032.5, 2147857974.5, 2147980509.1]",
+302,CC_scale,"[2.01373417518e-07, 1.9315808239e-07, 2.0030879899e-07, 2.00056479715e-07, 1.83693724568e-07, 2.11280599416e-07, 209272490423e-07]",
```

Comment:
Missing decimal point in the final value of the CC_scale coefficient.

Reply:
Fixed cc_scale coefficient w/ missing decimal point.